docs/quickstart/classification.md

```diff
@@ -0,0 +1,141 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Sklearn-evaluation quickstart
+
+In this guid we'll show how to compare and evaluate models with sklearn-evaluation.
```

Comment:
pass this through a spell check since there are a few typos. I saw "guid", and "paraters"